src/DataCollector/HttpDataCollector.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
`declare(strict_types=1);` - please remove this line everywhere because it breaks backward compatibility. We can discuss this change in context of v8.

Reply:
this change still exists @slash3b

Reply:
oops, sorry, will nuke it tonight